### ISSUE TITLE
Stuck on start screen, doesn't respond to touch screen 

### ISSUE THREAD

**Issue by hoss62** [luckydog7/Funkin-android#51]
I'm using Samsung Galaxy S10e, and the app doesn't respond to any screen touches. Please fix this

**Comment by phurgeon** [comment]
I have this same issue on my galaxy s20, and I cant get past the title unless I connect a key board.

**Comment by hoss62** [comment]
> I have this same issue on my galaxy s20, and I cant get past the title unless I connect a key board.

How do you connect a keyboard?

**Comment by Suhtther** [comment]
It's an easy fix it's with the recent Android update but next to the back button there should be a person click on that and it should work👍

**Comment by VitalieW0WKid** [comment]
I have the same problem,well I downloaded gamepad and I did pass the screen,but when I got to the menu the touch screen and the keyboard didnt responded,its possible because are with the touch Pen thing, I tried on my Motorola phone and it worked,same with my other Samsung tablet

**Comment by NikoRueWasTaken** [comment]
Yeah I have the same problem. Did anyone test of its just happens in this update?

**Comment by DanisLixo** [comment]
> Yeah I have the same problem. Did anyone test of its just happens in this update?

optimization problem
And it's not the version, It is the amount of file within the game

**Comment by NikoRueWasTaken** [comment]
> optimization problem
> And it's not the version, It is the amount of file within the game

Huh weird, it used to work. Maybe an update for the phone itself. Unless that's what you ment by optimization

**Comment by hoss62** [comment]
> It's an easy fix it's with the recent Android update but next to the back button there should be a person click on that and it should work👍

What person? I'm confused
